runtime/frequency/src/development_genesis.rs

```diff
@@ -0,0 +1,156 @@
+use common_primitives::node::Verify;
+use cumulus_primitives_core::ParaId;
+use sp_core::{sr25519, Pair, Public};
+use sp_runtime::traits::IdentifyAccount;
+#[cfg(not(feature = "std"))]
+use sp_std::alloc::format;
+
+use crate::*;
+
+// Generate the session keys from individual elements.
+//
+// The input must be a tuple of individual keys (a single arg for now since we have just one key).
+fn template_session_keys(keys: AuraId) -> SessionKeys {
+	SessionKeys { aura: keys }
+}
+
+#[allow(clippy::unwrap_used)]
+fn load_genesis_schemas() -> Vec<crate::pallet_schemas::GenesisSchema> {
+	serde_json::from_slice(include_bytes!("../../../resources/genesis-schemas.json")).unwrap()
+}
+
+type AccountPublic = <Signature as Verify>::Signer;
+
+pub fn development_genesis(
+	invulnerables: Vec<(AccountId, AuraId)>,
+	endowed_accounts: Vec<AccountId>,
+	council_members: Vec<AccountId>,
+	technical_committee_members: Vec<AccountId>,
+	id: ParaId,
+) -> serde_json::Value {
+	let genesis = RuntimeGenesisConfig {
+		system: Default::default(),
+		balances: BalancesConfig {
+			balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(),
+		},
+		parachain_info: ParachainInfoConfig { parachain_id: id, ..Default::default() },
+		collator_selection: CollatorSelectionConfig {
+			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
+			candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
+			..Default::default()
+		},
+		session: SessionConfig {
+			keys: invulnerables
+				.into_iter()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                 // account id
+						acc,                         // validator id
+						template_session_keys(aura), // session keys
+					)
+				})
+				.collect(),
+		},
+		// no need to pass anything to aura, in fact it will panic if we do. Session will take care
+		// of this.
+		aura: Default::default(),
+		aura_ext: Default::default(),
+		#[cfg(any(not(feature = "frequency-no-relay"), feature = "frequency-lint-check"))]
```

Comment:
having `any` before `not` is a bit weird. Should it be the other way around?

Reply:
I think I have a bad merge. I switched the PR to draft in order to fix it.